crengine/src/private/lvfreetypeface.cpp

```diff
@@ -1471,13 +1493,13 @@ lUInt16 LVFreeTypeFace::measureText(const lChar32 *text,
                             // And go on with the found glyph now that we fixed what was before
                         }
                         // Glyph found in this font
-                        advance = FONT_METRIC_TO_PX(glyph_pos[hg].x_advance);
+                        advance = FONT_METRIC_TO_PX(glyph_pos[hg].x_advance + _synth_weight_advance_add);
```

Comment:
Feels like this will be bad with diacritics (hg are glyphs, there can be multiple glyphs for a single cluster hcl), I guess we can have x_advance=0

Reply:
Yes, this is probably a weak point.

Reply:
Apply scaling factor to character width? This will likely slow down significantly...